test/functional/terminal/ex_terminal_spec.lua

```diff
@@ -105,9 +105,15 @@ describe(':terminal (with fake shell)', function()
 
   it('executes a given command through the shell', function()
     terminal_with_fake_shell('echo hi')
-    screen:expect([[
+    local escaped = [[
```

Comment:
Instead of this, can we force the old `shellxquote` for these tests? Because it's not important for these tests to test the default value of `shellxquote`. Then we avoid lots of churn and also makes it easier to rollback if the new default doesn't work out for some reason.

Reply:
Yes, because the affected test commands do not require any non-trivial escaping and have 0-1 double quotes. The new `shellxquote` is for commands with 2+ double quotes.